### ISSUE TITLE
Disabling privacy plugin throws an error

### ISSUE THREAD

**Issue by dickermoshe** [squidfunk/mkdocs-material#7624]
### Bug description

If you define that privacy is explicitly disabled, mkdocs serve crashes:

```yaml
plugins:
  - privacy:
      enabled: false
```

Error:
```console
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/mkdocs/livereload/__init__.py", line 211, in _build_loop
    self.builder()
  File "/usr/local/lib/python3.11/site-packages/mkdocs/commands/serve.py", line 67, in builder
    build(config, serve_url=None if is_clean else serve_url, dirty=is_dirty)
  File "/usr/local/lib/python3.11/site-packages/mkdocs/commands/build.py", line 319, in build
    env = config.plugins.on_env(env, config=config, files=files)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/mkdocs/plugins.py", line 599, in on_env
    return self.run_event('env', env, config=config, files=files)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/mkdocs/plugins.py", line 566, in run_event
    result = method(item, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/material/plugins/privacy/plugin.py", line 152, in on_env
    wait(self.pool_jobs)
         ^^^^^^^^^^^^^^
AttributeError: 'PrivacyPlugin' object has no attribute 'pool_jobs'
```

### Related links

- https://squidfunk.github.io/mkdocs-material/plugins/privacy/#config.enabled

### Reproduction

[9.5.41-explicit-disable-privacy.zip](https://github.com/user-attachments/files/17384760/9.5.41-explicit-disable-privacy.zip)

### Steps to reproduce

1. `mkdocs serve`
2. Crashes instantly

### Browser

Chrome

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) using the [built-in info plugin](https://squidfunk.github.io/mkdocs-material/plugins/info/).

**Comment by squidfunk** [comment]
Thanks for reporting. It's a 9.5.41 regression. Fixed in e2c621f1d.

**Comment by dickermoshe** [comment]
Thanks!
What is your release cycle? When will I see this on docker hub?

**Comment by squidfunk** [comment]
Yes, it will be part of 9.5.42. [Usually once a week](https://squidfunk.github.io/mkdocs-material/changelog/#material-for-mkdocs). We just released 9.5.41 yesterday, and this is not a fatal bug, so give it a few days. You can always use Material for MkDocs from git, to use the version that contains the fix.

**Comment by squidfunk** [comment]
Released as part of [9.5.42](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.42).
